.goreleaser.yml

```diff
@@ -178,6 +178,33 @@ builds:
       post:
         - cmd: ./lib/osslsigncode sign -n "Confluent CLI" -i "https://confluent.io" -pkcs12 CLIEVCodeSigningCertificate2.pfx -in {{ .Path }} -out {{ .Path }}
         - cmd: rm CLIEVCodeSigningCertificate2.pfx
+  - id: confluent-windows-amd64-chocolatey
```

Comment:
We might want to come up with a more generic name, i.e. `confluent-windows-amd64-noupdate` in case we want to support `scoop` (or other Windows package managers) in the future. We should probably consider doing the same for the linux/darwin builds too.